docs/source/faqs.rst

```diff
@@ -87,6 +108,20 @@ Collision integrals
     thermal masses (for the out-of-equilibrium particles) and momentum grid size as the model
     with which you obtained the collision integrals.
 
+- **WallGo tells me that it can not read the collision files.**
+
+    This might happen when you download the collision files from the git repository. 
+    As the git repository uses `git lfs` to manage the large collision files, the downloaded
+    files will be pointers, and not the full collision files.
+    You can obtain the collision files by cloning the repository, and running
+    
+    .. code-block:: bash
+
+        git install lfs
```

Comment:
I'm not sure this is correct. First you need to install Git Large File Storage. On a Mac you can do this with `brew install git-lfs` and on Ubunutu you could do `sudo apt-get install git-lfs`. Once installed, you can confirm the installation is working with git by running `git lfs install` not `git install lfs`.

Reply:
You could point the user to: https://docs.github.com/en/enterprise-server@3.17/repositories/working-with-files/managing-large-files/installing-git-large-file-storage?platform=windows using the .rst urls.